sass/layout/_sidebar-left.scss

```diff
@@ -595,18 +672,25 @@
         /* box-shadow: 0px 3px 4px 0px rgba(0, 0, 0, 0.33);
         transition: box-shadow 0.25s ease-in-out; */
     }
-    .SidebarNavContainer:not(.disabled) .SidebarChannelGroup .SidebarChannelGroupHeader_groupButton:hover {
-        cursor: pointer;
 
+    .SidebarNavContainer:not(.disabled) .SidebarChannelGroup .SidebarChannelGroupHeader:hover {
         .SidebarMenu {
             display: block;
         }
 
+        .SidebarCategory_newLabel {
+            display: none;
+        }
+
         .SidebarChannelGroupHeader_sortButton {
             display: block;
         }
+    }
 
-        > div, > i {
+    .SidebarNavContainer:not(.disabled) .SidebarChannelGroup:not(.dropDisabled) .SidebarChannelGroupHeader_groupButton:hover {
+        cursor: pointer;
```

Comment:
I noticed when playing with this on the test server that it turns the cursor to the pointer when you're dragging a channel over a not-disabled category. Do we even need to specify that the cursor is a pointer as long as we're using `button` or `a` for all the clickable elements?

Reply:
Yeah, looks like this was just done for good measure, came from the figma specs. Removed since it's not needed.